lib/device/lib/helpers.js

```diff
@@ -42,7 +43,7 @@ var readFileFromZip = function(zipPath, filePath) {
 
     fs
       .createReadStream(zipPath)
-      .pipe(unzip.Parse())
+      .pipe(unzip.Parse()) // eslint-disable-line
```

Comment:
Yessir:   46:19  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap